.github/workflows/CI.yml

```diff
@@ -1191,6 +1191,21 @@ jobs:
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
           aws-region: us-west-2
 
+      - name: Login to ECR
+        uses: docker/login-action@v1
+        with:
+          registry: {{ secrets.INTEG_TEST_ACCOUNT }}.dkr.ecr.us-east-1.amazonaws.com
+          username: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          password: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+        env:
+          AWS_REGION: us-east-1
+
+      - name: Pull image from public ECR and upload to Private ECR Repository
+        uses: akhilerm/tag-push-action@v2.0.0
```

Comment:
Use crane to copy images to not lose multiarch

https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docker-test-containers-daily.yml#L33

Reply:
`tag-push-action` uses crane. See doc for [action](https://github.com/akhilerm/tag-push-action) here.

Reply:
Although moving away from using vended GitHub actions where we can wouldn't be a bad idea.